prac_05/hex_colours.py

```diff
@@ -0,0 +1,24 @@
+def main():
+    colour_to_hex = {"Absolute Zero": "#0048ba",
+                     "Acid Green": "#b0bf1a",
+                     "Alizarin crimson": "#e32636",
+                     "Amaranth": "#e52b50",
+                     "Amber": "#ffbf00",
+                     "Amethyst": "#9966cc",
+                     "AntiqueWhite": "#faebd7",
+                     "Azure3": "#c1cdcd",
+                     "Azure4": "#838b8b",
+                     "Baby Blue": "#89cff0",
+                     "Baby Pink": "#f4c2c2",
+                     "Baker-Miller Pink": "#ff91af",
+                     "Banana Mania": "#fae7b5",
+                     "Banana Yellow": "#ffe135",
+                     "Barn Red": "#7c0a02"}
+    colour = input("colour: ")
+    while colour != "":
+        try:
```

Comment:
Should add something like: 
if colour in colour_to_hex:
       print("The colour code for \"{colour}\" is {colour_to_hex.get(colour)}")